dts/bindings/sensor/invensense,icm4268x-common.yaml

```diff
@@ -0,0 +1,356 @@
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    Generic bindings file for ICM-4268X models of motion tracking devices.
+    When setting the enum properties in a .dts or .dtsi file you may
+    include icm4268x.h and use the macros defined there.
+
+    Example:
+    #include <zephyr/dt-bindings/sensor/icm4268x.h>
+
+    icm42688: icm42688@0 {
+      ...
+
+      accel-pwr-mode = <ICM4268X_DT_ACCEL_LN>;
+      accel-fs = <ICM42688_DT_ACCEL_FS_16>;
+      accel-odr = <ICM4268X_DT_ACCEL_ODR_2000>;
+      gyro-pwr-mode= <ICM4268X_DT_GYRO_LN>;
+      gyro-fs = <ICM42688_DT_GYRO_FS_2000>;
+      gyro-odr = <ICM4268X_DT_GYRO_ODR_2000>;
+      accel-aaf-bw = <ICM4268X_DT_ACCEL_AAF_42HZ>;
+      gyro-aaf-bw = <ICM4268X_DT_GYRO_AAF_42HZ>;
+    };
+
+    Where,
+    ICM4268X_DT_ACCEL_ODR_2000 represents 2kHz as rate of sample capture by the
+    accelerometer.
+
+    ICM4268X_DT_GYRO_ODR_2000 represents 2kHz as rate of sample capture by the
+    gyroscope.
+
+    ICM4268X_DT_ACCEL_AAF_42HZ represents 42kHz as the bandwidth of the
+    Anti Alias Filter for the Accelerometer device.
+
+    ICM4268X_DT_GYRO_AAF_42HZ represents 42kHz as the bandwidth of the
+    Anti Alias Filter for the Gyroscope device.
+
+compatible: "invensense,icm4268x-common"
+include: [sensor-device.yaml, spi-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      This property describes the GPIO specification for INT line from ICM4268x package
+      on to your device. The property values should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  int1-pol:
+    type: int
+    default: 0
+    description: |
+      Choose if the interrupt1 polarity will be Active High or Ative Low.
+      Default polarity is as per datasheet section 14.3.
+    enum:
+      - 0 # ICM4268X_DT_INT1_POLARITY_ACTIVE_LOW
+      - 1 # ICM4268X_DT_INT1_POLARITY_ACTIVE_HIGH
+
+  int1-mode:
+    type: int
+    default: 0
+    description: |
+      Choose if the interrupt will be automatically latched and then cleared (pulsed mode)
+      Or latch and clear happens only when user code requests the data (latched mode).
+    enum:
+      - 0 # ICM4268X_DT_INT1_MODE_PULSED
+      - 1 # ICM4268X_DT_INT1_MODE_LATCHED
+
+  fifo-mode:
+    type: int
+    default: 0
+    description: |
+      Select what to do when FIFO is full. Stream to FIFO mode will overwrite LRU frame.
+      Stop on Full mode will prevent writing any frame if both FIFO and cache are full.
+      Default value is set as per the datasheet section 14.4.
+    enum:
+      - 0 # ICM4268X_DT_FIFO_MODE_BYPASS
+      - 1 # ICM4268X_DT_FIFO_MODE_STREAM
+      - 3 # ICM4268X_DT_FIFO_MODE_STOP_ON_FULL
+
+  fifo-ths-int-clr:
+    type: int
+    default: 1
+    description: |
+      Fifo threshold Interrupt Clearing Options.
+      Default value is set as per the datasheet section 14.49.
```

Comment:
all of these "default value set as per the datasheet" is not an actual explanation, it is something, and it can still mention the datasheet section, but also need to clearly say what is the reason for the value in this description, not just make the user go to investigate on their own. The point of the descriptions is to be descriptive.

Reply:
@decsny .. Changed.. please check